zcash_client_sqlite/src/wallet/init.rs

```diff
@@ -310,11 +533,365 @@ mod tests {
 
     use super::{init_accounts_table, init_blocks_table, init_wallet_db};
 
+    #[test]
+    fn init_migrate_from_0_3_0() {
```

Comment:
I have not checked that these tests initialize their respective db states correctly.